content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/differences.rst

```diff
@@ -2,46 +2,92 @@
+similarities between the two traceability methods, there are also notable differences that affect
+receipts, deliveries, and inventory reports.
+
+A lot usually indicates a specific batch of an item that was received, is currently stored, or was
+shipped from a warehouse, but can also pertain to a batch of products manufactured in-house, as
+well.
 
-In Odoo, lots and serial numbers have similarities in their functional
-system but are different in their behavior. They are both managed within
-the **Inventory**, **Purchases** and **Sales** app.
+A serial number is a unique identifier assigned incrementally (or sequentially) to an item or
+product, used to distinguish it from other items and products.
 
-**Lots** correspond to a certain number of products you received and store
-altogether in one single pack.
+.. seealso::
+   - :doc:`/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots`
+   - :doc:`/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/serial_numbers`
 
-**Serial numbers** are identification numbers given to one product in
-particular, to allow to track the history of the item from reception to
-delivery and after-sales.
+Enable lots & serial numbers
+============================
 
-When to use
------------
+To track products using lots and serial numbers, the *Lots & Serial Numbers* feature must be
+enabled. Go to the :menuselection:`Inventory app --> Configuration --> Settings`, scroll down to the
+:guilabel:`Traceability` section, and click the box next to :guilabel:`Lots & Serial Numbers`. Then,
+click the :guilabel:`Save` button to save changes.
 
-**Lots** are interesting for products you receive in great quantity and for
-which a lot number can help in reportings, quality controls, or any
-other info. Lots will help identify a number of pieces having for
-instance a production fault. It can be useful for a batch production of
-clothes or food.
+.. image:: differences/differences-enabled-setting.png
+   :align: center
+   :alt: Enabled lots and serial numbers feature in inventory settings.
 
-**Serial numbers** are interesting for items that could require after-sales
-service, such as smartphones, laptops, fridges, and any electronic
-devices. You could use the manufacturer's serial number or your own,
-depending on the way you manage these products
+When to use lots
+================
 
-When not to use
----------------
+Lots are useful for products that are manufactured or received in large quantities (such as clothes
+or food), and can be used to trace a product back to a group. This is especially useful when
+managing product recalls or expiration dates.
 
-Storing consumable products such as kitchen roll, toilet paper, pens and
-paper blocks in lots would make no sense at all, as there are very few
-chances that you can return them for production fault.
+.. image:: differences/differences-lot.png
+   :align: center
+   :alt: Created lot with quantity of products in it.
```

Comment:
What do you think about putting this in an example block? Also how did you navigate to this page? If the instructions are included in another doc, maybe we can link that instead!